Automattic/jetpack
# storybook: Remove unneeded dependencies

#### Changes proposed in this Pull Request:

There are a lot of dependencies in js-packages/storybook that don't seem
to be used. Modulo a few cases where a minifier seems to make different
choices (e.g. whether it outputs `calc((2/3) * 1em)` or `.6666666667em`)
we get the same output with these packages removed:

- @automattic/jetpack-base-styles (#23386 removed the uses of it)
- @babel/runtime-corejs3
- @babel/traverse
- @emotion/cache
- @emotion/css
- @emotion/jest
- @emotion/native
- @emotion/utils

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Check that the build artifact for Automattic/jetpack-storybook doesn't contain any unexpected changes.
